NOM_visualizations/python/nom_data.ipynb

```diff
@@ -0,0 +1,3353 @@
+{
```

Comment:
<p>I left a comment on Bea's notebook too, so maybe you guys should discuss. Your code assumes that the input into the <code>omics_processing_set</code> will be a Biosample, but the schema also allows for <code>ProcessedSample</code>s to be an input. This can get really messy (look at the taxonomy notebook). But because you aren't checking for processed samples, you may be missing data. I think we can leave this alone for this notebook. I believe the only study that takes advantage of the ProcessedSamples is NEON. So if there are NEON samples that went through NOM, your notebook would be missing that data. We can discuss in tomorrow's meeting, but hopefully there aren't NEON samples, or maybe we can just ignore this for now since we already have a notebook that shows that really crazy schema traversal.</p><p> </p><p>One thing we could do which Bea mentioned is to use this endpoint: <a href="https://api.microbiomedata.org/docs#/metadata/get_collection_name_by_doc_id_nmdcschema_ids__doc_id__collection_name_get" target="_blank">https://api.microbiomedata.org/docs#/metadata/get_collection_name_by_doc_id_nmdcschema_ids__doc_id__collection_name_get</a> to get the collection names that the <code>omice_processing_set</code> <code>has_input</code> belongs to. If there are not <code>ProcessedSamples</code>. and just <code>Biosamples</code>. Then we're golden.</p><p> </p><p>Maybe we can also assume there are no ProcessedSamples because your code didn't break in the next API call?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>